contracts/LoanManager.sol

```diff
@@ -19,34 +19,34 @@ import "./MonetarySupervisor.sol";
 contract LoanManager is Restricted, TokenReceiver {
     using SafeMath for uint256;
 
-    enum LoanState { Open, Repaid, Defaulted, Collected } // NB: Defaulted state is not stored, only getters calculate
+    enum LoanState { Open, Repaid, DoNotUse, Collected } // NB: DoNotUse state is kept for backwards compatibility only (so the ordinal of 'Collected' does not shift), as the name states: do not use it.
 
     struct LoanProduct {
-        uint minDisbursedAmount; // 0: with decimals set in AugmintToken.decimals
-        uint32 term;            // 1
-        uint32 discountRate;    // 2: discountRate in parts per million , ie. 10,000 = 1%
-        uint32 collateralRatio; // 3: loan token amount / colleteral pegged ccy value
-                                //      in parts per million , ie. 10,000 = 1%
-        uint32 defaultingFeePt; // 4: % of collateral in parts per million , ie. 50,000 = 5%
-        bool isActive;          // 5
+        uint minDisbursedAmount;    // 0: minimum loanAmount, with decimals set in AugmintToken.decimals (i.e. token amount)
+        uint32 term;                // 1: term length (in seconds)
+        uint32 discountRate;        // 2: discountRate (in parts per million, i.e. 10,000 = 1%)
+        uint32 collateralRatio;     // 3: inverse of collateral ratio: [repayment value (in token) / collateral value (in token)] (in ppm). Note: this is actually the inverse of the commonly used "collateral ratio"! TODO: fix it
+        uint32 defaultingFeePt;     // 4: % of repaymentAmount (in parts per million, i.e. 50,000 = 5%)
+        bool isActive;              // 5: flag to enable/disable product
+        uint32 minCollateralRatio;  // 6: minimum collateral ratio: [collateral value (in token) / repayment value (in token)] (in ppm), defines the margin, zero means no margin. Note: this is _not_ an inverse like the above "collateralRatio", it is already stored properly!
```

Comment:
Wasn't the plan to call the new prop `marginCallRate` and rename existing`collateralRatio` prop to `minColleteralRatio`? Not too fussy but it's confusing in this current form.
How about `minInitialCollateralRatio` and `marginCallColletaralRatio?

Reply:
What’s formerly known as `collateralRatio` should really be `initialCollateralRatio` (and inverted to fit how the term is commonly used). An aside: isn’t this defined by the loan product?

The new one is aptly called the minimum (position can be liquidated below this collateralization ratio).

Reply:
I've no strong opinion on this.